lib/broccoli/filter-translation-tree.js

```diff
@@ -0,0 +1,22 @@
+const funnel = require('broccoli-funnel');
+
+function filterTranslationTree(tree, includeLocales, excludeLocales) {
```

Comment:
Can we add tests for this

Reply:
https://github.com/ember-intl/ember-intl/blob/master/tests-node/unit/broccoli/translation-reducer/index-test.js